 syntax = "proto3";

 package rl_trainer;

 message ExperienceTuple {
     repeated double state_1 = 1;
     repeated double action = 2;
     double reward = 3;
     repeated double state_2 = 4;
     bool state_2_is_terminal = 5;
 }

 message Episode {
     repeated ExperienceTuple experience_tuples = 1;
 }